src/test/fuzz/secp256k1_ec_seckey_import_export_der.cpp

```diff
@@ -20,7 +20,7 @@ FUZZ_TARGET(secp256k1_ec_seckey_import_export_der)
     secp256k1_context* secp256k1_context_sign = secp256k1_context_create(SECP256K1_CONTEXT_NONE);
```

Comment:
if we still need this, we can move it to the scope where it's used (with the secp256k1_context_destroy call)

Reply:
Makes sense yes, done. Packed the scope reduction in the same commit as I think the diff is still digestible, but happy to divide it into two commits if that's preferable for reviewers.